okokyst_map.py

```diff
@@ -1,5 +1,8 @@
 import matplotlib
-from mpl_toolkits.basemap import Basemap
```

Comment:
I think we should remove all basemap references and replace with cartopy

Reply:
In general yes, but I have not been looking at all to the maps yet. And have not been really planning to, but we can discuss it :) I just here added 'try and except' for basemap importing, because the whole program was failing because of this import.

Reply:
Yes I understand this is a bigger undertaking, although it is mostly a few plots. Would you be willing to do the job? I can ask Andrew for some extra GB hours/days for you if you like